docs/processes/project-lifecycle.md

```diff
@@ -22,7 +22,7 @@ Find the template: [Idea Draft](https://www.notion.so/Idea-Draft-1859a4c092c780a
 
 Once the Idea Draft is approved, the Partner Lead, Team Lead and Architect will move on to the Tech Design. This document provides an overview of each contract, external function, and off-chain component, defining how every piece of functionality works and outlining a time estimate. To mitigate risks, we also break the project into smaller milestones—each with its own timeline and deliverables—to keep everyone aligned and help achieve the partner’s goals efficiently.
 
-While the project is still being defined and refined, we start preparing the advanced tests, by collecting invariants. This initial collection is often a first draft, as many of these might still change during the development.
+While the project is still being defined and refined, we start preparing the advanced tests, by collecting invariants. This initial collection is often a first draft, as many of these might still change during the development. In addition, to ensure as must robustness as possible, the security department will review it to catch early problems.
```

Comment:
could we also clarify that the property definition and the security review should not be a blocker to development, but done as early as possible, in case it affects the arquitecture?

Let's try to also explain the reasoning behind our decisions whenever possible